config/action.d/badips.py

```diff
@@ -80,13 +82,14 @@ class BadIPsAction(ActionBase):
 	"""
 
 	_badips = "http://www.badips.com"
-	_Request = partial(
-		Request, headers={'User-Agent': "Fail2Ban %s" % f2bVersion})
+	def _Request(self, url, **argv):
+		return Request(url, headers={'User-Agent': self.agent}, **argv)
```

Comment:
sorry -- got confused by the positioning of `__doc__` here so diff looked like the body of `__init__` to me (despite it below ;) )